src/Servers/Kestrel/Kestrel/src/WebHostBuilderKestrelExtensions.cs

```diff
@@ -29,6 +29,7 @@ public static class WebHostBuilderKestrelExtensions
         /// </returns>
         public static IWebHostBuilder UseKestrel(this IWebHostBuilder hostBuilder)
         {
+            hostBuilder.UseQuic();
```

Comment:
So anyone using Kestrel will be using Quic, even if not using Http3? Does it no-op for Http1/2?

Reply:
Right, this adds the services but they're only used if you enable http3.